Geometry/test/testGeometryTestBeam.py

```diff
@@ -13,6 +13,8 @@
 #process.load('Configuration.StandardSequences.RecoSim_cff')
 #process.load('Configuration.StandardSequences.EndOfProcess_cff')
 
+process.TFileService = cms.Service("TFileService",fileName = cms.string("histo.root"))
+
```

Comment:
why is the histo added?

Reply:
Ah, it's for the visualization that I use. I will remove that line soon.

Reply:
but u dont run that here

Reply:
I've added the TFile output in my GEMGeometryAnalyser.
It generates the 3d histogram with strip positions.
Anyway, it will be removed in next push.